src/actions/index.js

```diff
@@ -56,6 +60,21 @@ export const addProject = (project: Project) => ({
   project,
 });
 
+export const showDeleteProjectPrompt = (project: Project) => ({
+  type: SHOW_DELETE_PROJECT_PROMPT,
+  project,
+});
+
+export const deleteProjectFromDisk = (project: string) => ({
+  type: DELETE_PROJECT_FROM_DISK,
+  project,
+});
+
+export const deleteProjectFromDiskFinish = (deletedProject: string) => ({
+  type: FINISH_DELETING_PROJECT_FROM_DISK,
```

Comment:
Looks like there's an inconsistency between the action-creator name and the action constant.

Since I think we should kill `DELETE_PROJECT_FROM_DISK`, I quite like `FINISH_DELETING_PROJECT_FROM_DISK`; I think we should update the action-creator name to be the same :)